.github/workflows/gradle.yml

```diff
@@ -22,6 +22,8 @@ jobs:
 
     steps:
     - uses: actions/checkout@v4
+      with:
```

Comment:
self-review: need the CI to also pull in the `partiql-lang-kotlin` git submodule